charts/tenantfilesystem/templates/env-xrd.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apiextensions.crossplane.io/v1
```

Comment:
Hi @crimbler2, 

The Env XRD is required to copy&paste EnvironmentConfig variables into the Composit Ressources properly. Because it is not possible to use CombineFromComposite patches with EnvironmentConfig Variables.